ROADMAP.md

```diff
@@ -8,19 +8,15 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.16
+## Rook Ceph 1.17
 
-The following high level features are targeted for Rook v1.16 (December 2024). For more detailed project tracking see the [v1.16 board](https://github.com/orgs/rook/projects/6).
+The following high level features are targeted for Rook v1.17 (April 2025). For more detailed project tracking see the [v1.17 board](https://github.com/orgs/rook/projects/7).
 
-* Removed support for Ceph Quincy since at end of life [#14795](https://github.com/rook/rook/pull/14795)
-* Enable mirroring for RADOS namespaces [#14701](https://github.com/rook/rook/pull/14701)
+* Allow Rook operator to run in multiple namespaces for improved multi-cluster reconcile [#15014](https://github.com/rook/rook/issues/15014)
 * Replace a single OSD when a metadataDevice is configured with multiple OSDs [#13240](https://github.com/rook/rook/issues/13240)
```

Comment:
we can still keep the,

* Key rotation for Ceph object store users [#11563](https://github.com/rook/rook/issues/11563)

As it still needs to work upon

Reply:
Actually we were discussing in the community meeting that it is planned to be worked on soon and backported, so we are actually hoping for it still in 1.16 and no need for it in this list.